### PR TITLE
Update python examples

### PR DESCRIPTION
**Pull request by VHolonec**

### CODE REVIEW

File: bindings/python/examples/data_collect/data_collect.py [hunk 1]
```diff
@@ -179,9 +179,9 @@
             print('Please reboot the board')
 
     #Get frame types
-    frame_types = []
-    status = camera1.getAvailableFrameTypes(frame_types)
-    if status != tof.Status.Ok or len(frame_types) == 0:
+    modes = []
+    status = camera1.getAvailableModes(modes)
+    if status != tof.Status.Ok or len(modes) == 0:
         sys.exit('Cound not aquire frame types')
```
Comment (dNechita) [inline bindings/python/examples/data_collect/data_collect.py:185]: Could not acquire available modes

File: bindings/python/examples/first_frame/first_frame.py [hunk 2]
```diff
@@ -106,17 +106,17 @@ def callbackFunction(callbackStatus):
...
 
 camDetails = tof.CameraDetails()
 status = camera1.getDetails(camDetails)
 print("camera1.getDetails()", status)
 print("camera1 details:", "id:", camDetails.cameraId, "connection:", camDetails.connection)
 
-status = camera1.setFrameType("lr-qnative")
+status = camera1.setMode("lr-qnative")
 print("camera1.setFrameType()", status)
 print("lrqmp")
```
Comment (dNechita) [inline bindings/python/examples/first_frame/first_frame.py:121]: While at it, update "lrqmp" to "lr-qnative"

**Review by dNechita — APPROVED** [review]
Looks good!

File: bindings/python/examples/dnn/dnn.py [hunk 3]
```diff
@@ -107,11 +107,6 @@ class ModesEnum(Enum):
     status = camera1.setMode(modes[ModesEnum.MODE_NEAR.value])
     if not status:
         print("camera1.setMode() failed with status: ", status)
-
-    types = []
-    status = camera1.getAvailableFrameTypes(types)
-    if not status:
-        print("system.getAvailableFrameTypes() failed with status: ", status)
         
     status = camera1.setFrameType(mode)
```
Comment (SeptimiuVana) [inline bindings/python/examples/dnn/dnn.py:111]: This needs to be updated.

File: bindings/python/examples/dnn/dnn.py [hunk 4]
```diff
@@ -107,11 +107,6 @@ class ModesEnum(Enum):
     status = camera1.setMode(modes[ModesEnum.MODE_NEAR.value])
```
Comment (SeptimiuVana) [inline bindings/python/examples/dnn/dnn.py:107]: This one is using a deprecated modesEnum structure. Needs to be updated as well.

File: bindings/python/examples/maskr_cnn/maskr_cnn.py [hunk 5]
```diff
@@ -189,14 +189,9 @@ def get_scaling_values(cameraDetails):
...
+    status = cameras[0].setMode(modes[0])
     if not status:
-        print("system.getAvailableFrameTypes() failed with status: ", status)
-
-    status = cameras[0].setFrameType(types[0])
-    if not status:
-        print("cameras[0].setFrameType() failed with status:", status)
+        print("cameras[0].setMode() failed with status:", status)
 
     status = cameras[0].setMode(modes[ModesEnum.MODE_NEAR.value])
```
Comment (SeptimiuVana) [inline bindings/python/examples/maskr_cnn/maskr_cnn.py:196]: This needs to be updated.

File: bindings/python/examples/skeletal_tracking/skeletal_tracking.py [hunk 6]
```diff
@@ -90,15 +90,10 @@ class ModesEnum(Enum):
     status = camera1.setMode(modes[ModesEnum.MODE_NEAR.value])
```
Comment (SeptimiuVana) [inline bindings/python/examples/skeletal_tracking/skeletal_tracking.py:90]: This one too.

**Review by SeptimiuVana — APPROVED** [review]

**Pull request merged**